### ISSUE TITLE
Project dependencies with the .pulldownrc file

### ISSUE THREAD

**Issue by k33g** [jackfranklin/pulldown#6]
Hello,
it would be fine if it is possible to change the name of destination file, something like this

```
{
    "destination": "vendors",
    "dependencies": [
        {url:"http://code.jquery.com/jquery.min.js",name:"jquery"}
    ]
}
```

pulldown runs very well under windows 7 :)

**Comment by jackfranklin** [comment]
> pulldown runs very well under windows 7 :)

Firstly, that's great news! Thanks :)

I also agree with the idea that you should be able to edit the names within the `.pulldownrc` file, regardless of if you use the URL or named dependency.

I'm thinking something like this:

```
{
    "destination": "vendors",
    "dependencies": [
        { location :"http://code.jquery.com/jquery.min.js", output:"jquery.js"}
    ]
}
```

Where `location` is either a URL or a name to find in `~/.pulldownrc`.

What do you think to that? Thanks :)

**Comment by k33g** [comment]
it would be perfect :)

**Comment by jackfranklin** [comment]
Just fixed this with V0.1.2

**Comment by k33g** [comment]
I just tested on windows, it works great :) so cool!
